tests/functional/suite_test.go

```diff
@@ -68,7 +68,7 @@ const (
 	ContainerImage       = "test://nova"
 	timeout              = 40 * time.Second
```

Comment:
This is a fairly big number that slows down the test execution. But the deployment controller has a 15 seconds Requeue request at some point so this 40 sec is probably needed today. I encourage to a) check if Requeue is really needed there or it can be replaced with some watch functionality. b) if a) is not possible then I suggest to make that 15 seconds configurable so that we can override that from the test to be smaller.  These are obviously outside of the scope of the current PR.

Reply:
ACK